qutip/solver/ode/verner9efficient.pyx

```diff
@@ -0,0 +1,850 @@
+#cython: language_level=3
+# Verner 9 Efficient
+# http://people.math.sfu.ca/~jverner/
+import numpy as np
+cimport numpy as cnp
+from qutip.solver.ode.explicit_rk cimport Explicit_RungeKutta
+from .wrapper cimport QtOdeData, QtOdeFuncWrapper
+from .wrapper import qtodedata
```

Comment:
Why not use absolute imports? 

```
from qutip.solver.ode.wrapper ...
```

Reply:
To me this is the poster child for relative imports.  As far as I see, absolute style only has negatives: it would be much less clear how the module `wrapper` relates to this one (i.e. part of the same package), it adds maintenance cost if any part of the path ever changes (and lots of internal paths do change in `dev.major`), and it's generally just harder to scan the lines because there's more fluff.  PEP 8 suggests
> explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose

QuTiP's certainly got a complex package layout (this bit's nested 4 deep).

I agree with you on stuff like `from ...core` though - that should be `from qutip import x` (`core` is physical organisation only - it's transparent to the user).

When I separated out the core package, which modified every `import` statement, I generally tried to stick to "absolute when crossing an API boundary, relative within the same API implementation", which is why loads of `dev.major` swapped to relative imports.  I'm sure I didn't get it perfect, though.